### PR TITLE
Add domain operations in Graph rbac

### PR DESCRIPTION
**Pull request by jianghaolu**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by vishrutshah** [comment]
Hello @jianghaolu, Feel free to loop in the service team owners to make sure they are aware and to get their signoff on PR as well :)

**Comment by jianghaolu** [comment]
Hi @shuagarw , please advise if this change is acceptable to you. Thank you!

**Comment by fearthecowboy** [comment]
ATTENTION REQUIRED.
-------------

We have changed the `default` branch in the `azure-rest-api-specs` repository to use the branch labeled `current`.

This PR is opened against the `master` branch.

You have two options: 
1. Keep this PR open, and complete the review process as normal, and once this PR is merged against the `master` branch, we (@olydis & @fearthecowboy)  will create a PR against the current branch and make any updates required to work in the newly reorganized default branch.  

2. Close this PR and create a new one. That would mean pulling in the changes from the new default branch `current` moving any files and switching to the configuration-based layout.  (You are more than welcome to choose option 2, but I don't expect anyone to do that)

Either way, the work you are currently doing against the previous default branch `master` will be the last PR against that branch that we will accept and all new PRs will go against the new default branch  `current`.

If you have any questions or concerns, ping @fearthecowboy or @olydis (or go ask on the `#swagger` channel in slack) ASAP and we can help you thru the process.

**Comment by jianghaolu** [comment]
Breaking changes eliminated.

File: arm-graphrbac/1.6/swagger/graphrbac.json
Comment (darshanhs90) [inline arm-graphrbac/1.6/swagger/graphrbac.json]: >/{tenantID}/domains":  [](start = 5, length = 22)

will this endpoint be ported to powershell/nodejs?

File: arm-graphrbac/1.6/swagger/graphrbac.json
Comment (vishrutshah) [inline arm-graphrbac/1.6/swagger/graphrbac.json]: Seems like `Domain` is only used in the `DomainListResult` and in that case `"required"` should not be used for `name` and it'd be good to mark it as readOnly.

**Review by vishrutshah — CHANGES_REQUESTED** [review]
@jianghaolu  Other than one comment. change LGTM!

File: arm-graphrbac/1.6/swagger/graphrbac.json
Comment (jianghaolu) [inline arm-graphrbac/1.6/swagger/graphrbac.json]: I don't believe so - I added this because this is needed in the fluent SDK development. If it's needed in PowerShell / CLI they can surely take advantage of this.

File: arm-graphrbac/1.6/swagger/graphrbac.json
Comment (jianghaolu) [inline arm-graphrbac/1.6/swagger/graphrbac.json]: There's a REST API to create a domain - it was not needed so I didn't add it. What do you think?

Comment (vishrutshah) [inline arm-graphrbac/1.6/swagger/graphrbac.json]: Okay. I am fine in keeping it.

**Review by vishrutshah — APPROVED** [review]

**Pull request merged**
